sirepo/package_data/template/genesis/parameters.py.jinja

```diff
@@ -0,0 +1,14 @@
+# -*- python -*-
+import subprocess
+
+# blank line at end of file is required for genesis to
+# properly parse the file
+input_file_variables = """$newrun
+{{ variables }}
+$end
+"""
+
+with open('{{ input_filename }}', 'w') as f:
+    f.write(input_file_variables)
```

Comment:
@robnagler can we use pykern in parameters.py? It will eventually be downloadable for the user to run on their machine so maybe not to reduce things they have to install?